tar/bsdtar.c

```diff
@@ -1399,6 +1414,48 @@ configfile_helper(struct bsdtar *bsdtar, const char *line)
 	return (0);
 }
 
+static void
+archive_names(struct bsdtar *bsdtar, const char *fname)
+{
+	struct stat sb;
+
+	/* If it's a file, does it exist? */
+	if (strcmp(fname, "-") != 0) {
+		if (stat(fname, &sb))
```

Comment:
Why?  The only test which matters is whether `process_lines` can read the file, and it handles errors.